### PR TITLE
bugfix: wrap images correctly

### PR DESCRIPTION
**Pull request by ParthSareen**
There is currently an issue with image serialization and passing in Images as part of the message directly. Works off #386 shoutout to @akx!
Closes:
- https://github.com/ollama/ollama-python/pull/386
- https://github.com/ollama/ollama-python/issues/375

### CODE REVIEW

File: ollama/_client.py [hunk 1]
```diff
@@ -189,7 +189,7 @@ def generate(
     stream: Literal[False] = False,
     raw: bool = False,
     format: Optional[Union[Literal['', 'json'], JsonSchemaValue]] = None,
-    images: Optional[Sequence[Union[str, bytes]]] = None,
```
Comment (ParthSareen) [inline ollama/_client.py:192]: Going to revert this change to keep PR scope small for now

**Review by jmorganca — APPROVED** [review]
